framework/views/_addForeignKeys.php

```diff
@@ -1,4 +1,4 @@
-<?php foreach ($foreignKeys as $column => $fkData): ?>
+<?php foreach ($foreignKeys as $column => $fkData) : ?>
```

Comment:
This seems to be correct: https://github.com/squizlabs/PHP_CodeSniffer/issues/366#issuecomment-66937317

> I'm not sure what PSR2 actually intends in this case. They don't mention the alternate syntax anywhere. So I still need to enforce the spaces after opening parentheses error...